src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -64,6 +67,9 @@ const UserEnrollmentForm = ({
       enterpriseId,
       'edx.ui.enterprise.learner_portal.executive_education.checkout_form.submitted',
     );
+    await queryClient.invalidateQueries({
+      queryKey: enterpriseUserSubsidyQueryKeys.redeemablePolicies(enterpriseId, userId),
```

Comment:
[clarification/sanity check] Would we want/need to invalidate _all_ policy-related queries here (i.e., not just the list of redeemable policies)? For example, does the `can-redeem` API response also need to be invalidated here, where after enrollment the `can-redeem` may still be returning now-stale data?

Reply:
We should probably clear all the query keys here 👍🏽  Good call out